include/sound/sof/info.h

```diff
@@ -126,4 +127,11 @@ struct sof_ipc_probe_support {
 	uint32_t reserved[2];
 } __packed;
 
+/* extended data: user abi version(s) */
+struct sof_ipc_user_abi_version {
```

Comment:
"user abi"  is a bit weird name, but alas this was already merged to FW, so it's what it is.